pkg/apps/chart_app.go

```diff
@@ -13,6 +14,12 @@ func MakeInstallChart(options *types.InstallerOptions) (*types.InstallerOutput,
 		return nil, err
 	}
 
+	for _, secret := range options.Secrets {
```

Comment:
Do you want to exit? We used to warn when the secret existed

Reply:
This bubbles up the `secretErr` that we used to error on, when the command has an exit code != 0 we just print a log line
```go 
	res, secretErr := KubectlTask(args...)

	if secretErr != nil {
		return secretErr
	}
```